controllers/drplacementcontrol_controller.go

```diff
@@ -715,8 +706,15 @@ func (d *DRPCInstance) runInitialDeployment() (bool, error) {
 	// Make sure we record the state that we are deploying
 	d.setDRState(rmn.Deploying)
 
+	// Create VRG first, to leverage user PlacementRule decision to skip placement and move to cleanup
+	const restorePV = false
+	err := d.createVRGManifestWork(homeCluster, restorePV)
```

Comment:
Actually, we are waiting inside createVRGManifestWork(...).
As far as calling runPlacementTask(), yes, I agree, we should.  We just have to remove the cleanup() call from it, because, this is initial deployment and it does not need a cleanup.